src/feed/Feed.js

```diff
@@ -144,7 +144,11 @@ class Feed {
         const granularity = calculateGranularity(period, interval);
         const key = this._getKey({ symbol, granularity });
         const localDate = this._serverTime.getLocalDate();
-        suggestedStartDate = suggestedStartDate > localDate ? localDate : suggestedStartDate;
+        if (suggestedStartDate > localDate) {
+            suggestedStartDate = localDate;
+        } else if (!this.startEpoch && suggestedStartDate > this.endEpoch) {
+            suggestedStartDate = CIQ.strToDateTime(getUTCDate(this.endEpoch - 1000));
+        }
```

Comment:
Do we need to include `else` in this condition?

Reply:
I agree that we don't 👍🏻  Thank you)